packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.scss

```diff
@@ -39,8 +39,47 @@
   width: 7.063rem;
 }
 
-.vitalsSignsRadioButton {
-  @extend .bodyShort01;
-  color: $ui-05;
-  padding: $spacing-03 0;
+.verticalTabs {
+  margin: 1rem 0;
+  scroll-behavior: smooth;
+}
+
+.verticalTabs > ul {
+  flex-direction: column !important;
+}
+
+.tab:hover {
+  border-bottom: none !important;
+}
+
+.tab > button:first {
+  text-indent: 5rem;
+}
+
+.tab > button, 
```

Comment:
This file looks very similar...

Sure we cannot combine things here?

Reply:
Yes these two files are similar. Vitals and Biometrics are generally similar. I think combining them can be a future enhancement?

Reply:
Alright let's do it like that.